core/services/vrf/vrf_coordinator_solidity_crosscheck_test.go

```diff
@@ -59,18 +60,20 @@ func newIdentity(t *testing.T) *bind.TransactOpts {
 	return bind.NewKeyedTransactor(key)
 }
 
-// deployCoordinator sets up all identities and contracts associated with
+// newVRFCoordinatorUniverse sets up all identities and contracts associated with
```

Comment:
Nice rename. "Coordinator" was doing some heavy lifting, there are a lot of pieces that came along with it.